src/ui/pointcloud/qgspointcloudlayersaveasdialogbase.ui

```diff
@@ -0,0 +1,362 @@
+             <number>2</number>
+            </property>
+            <attribute name="verticalHeaderVisible">
+             <bool>false</bool>
+            </attribute>
+            <row/>
+            <column/>
+            <column/>
+           </widget>
+          </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_5">
+            <item>
+             <widget class="QPushButton" name="mSelectAllAttributes">
+              <property name="text">
+               <string>Select All</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QPushButton" name="mDeselectAllAttributes">
+              <property name="text">
+               <string>Deselect All</string>
+              </property>
+             </widget>
+            </item>
+           </layout>
+          </item>
+         </layout>
+        </widget>
+       </item>
+       <item>
+        <widget class="QgsExtentGroupBox" name="mExtentGroupBox">
+         <property name="title">
+          <string>Extent</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QgsCollapsibleGroupBox" name="mZRangeGroupBox">
+         <property name="title">
+          <string>Elevation Range</string>
+         </property>
+         <property name="checkable">
+          <bool>true</bool>
+         </property>
+         <property name="checked">
+          <bool>false</bool>
+         </property>
+         <layout class="QGridLayout" name="gridLayout">
+          <item row="2" column="0">
+           <widget class="QLabel" name="label_4">
+            <property name="text">
+             <string>Maximum Z value</string>
+            </property>
+           </widget>
+          </item>
+          <item row="2" column="1">
+           <widget class="QgsDoubleSpinBox" name="mMaximumZSpinBox"/>
+          </item>
+          <item row="1" column="1">
+           <widget class="QgsDoubleSpinBox" name="mMinimumZSpinBox"/>
+          </item>
+          <item row="1" column="0">
+           <widget class="QLabel" name="label_3">
+            <property name="text">
+             <string>Minimum Z value</string>
+            </property>
+           </widget>
+          </item>
+         </layout>
+        </widget>
+       </item>
+       <item>
+        <widget class="QgsCollapsibleGroupBox" name="mPointsLimitGroupBox">
+         <property name="title">
+          <string>Limit number of points</string>
+         </property>
+         <property name="checkable">
+          <bool>true</bool>
+         </property>
+         <property name="checked">
+          <bool>false</bool>
+         </property>
+         <layout class="QFormLayout" name="formLayout">
+          <item row="0" column="0">
+           <widget class="QLabel" name="label_5">
+            <property name="text">
+             <string>Limit total number of exported points to:</string>
```

Comment:
```suggestion
             <string>Limit total number of exported points to</string>
```
No colon on labels.
And unless there is a plan to add more entries, I was wondering whether we need that group box with single item (with almost the same label). [Not HIG compliant](https://docs.qgis.org/testing/en/docs/developers_guide/hig.html). What about:
* a single spinbox showing `no limit` and then people enter their value
![image](https://user-images.githubusercontent.com/7983394/183017340-3e039a54-775a-43a0-8a58-07f382fc5d0d.png)

* a checkbox followed by a spinbox, on the same line
![image](https://user-images.githubusercontent.com/7983394/183017203-1295d7f8-a0d8-4b74-9b78-4056041397bf.png)

We would probably then need to move that single option somewhere upper...

Reply:
There's a though to add a separate widget in that group that would limit the total number of points by stopping at a particular node level, resulting in a (kind of) predefined point density.

While groupboxes with single items do clutter the ui, imho in this specific case it makes sense, since it's on a list of collapsible-checkable groupboxes. Having one type of filter without a groupbox would make it look weirder and also make it stand-out like it's a more important setting.

Since the HIG says to *avoid* using those, I'd propose to keep it like that, if you do not insist!

Reply:
> There's a though to add a separate widget in that group that...

The you hit my "_unless there is a plan to add more entries_"
> Having one type of filter without a groupbox would make it look weirder and also make it stand-out like it's a more important setting.

Agreed. Reason why I was suggesting to move it somewhere else (no idea which other options it could have cohabited with though)
> Since the HIG says to avoid using those, I'd propose to keep it like that, if you do not insist!

No insistence. See above.